### PR TITLE
feat: ETCM-3002-sidechain-docs-release-1

### PR DESCRIPTION
**Pull request by stevanlohja**
This adds the first release of the sidechain docs.
ETCM-3002

### CODE REVIEW

**Review by drospa — APPROVED** [review]
Looks good to me

**Review by olgahryniuk — DISMISSED** [review]

**Review by olgahryniuk — DISMISSED** [review]

**Review by drospa — APPROVED** [review]
approved

File: content/08-cardano-sidechains/04-support/01-sidechain-faq.mdx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+### Will there be an alternative chain follower to DB Sync that consumes less resources?
+
+Yes, IOG is working in a less resource-intensive tool. There are other chain indexer tools such as [Oura](https://github.com/txpipe/oura) and [Carp](https://github.com/dcSpark/carp) that could serve the purpose of building a Cardano sidechain.
+
+## The example EVM sidechain application
+
+These questions relate to the application that IOG built as a proof of concept, using the Cardano Sidechains Toolkit.
+
+### What is the difference between the EVM sidechain design and Milkomeda?
```
[the remark this answers is not in the document]
Comment (olgahryniuk) [reply content/08-cardano-sidechains/04-support/01-sidechain-faq.mdx]: resolved by @stevanlohja

**Review by olgahryniuk — APPROVED** [review]

File: content/08-cardano-sidechains/02-sidechain-toolkit/03-chain-follower.mdx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+## Configuring the sidechain node to use your DB Sync instance
+
+After installing and setting up a DB Sync instance, you need to configure a sidechain node to point to it. In the node’s configuration file, set the data source type to `db-sync` and configure connection details to match your setup. The finished configuration should look similar to the example below:
+
+```
+{
+...
+
+  "mamba": {
```
Comment (stevanlohja) [inline content/08-cardano-sidechains/02-sidechain-toolkit/03-chain-follower.mdx:41]:
```suggestion
  "sc-client": {
```

**Review by olgahryniuk — DISMISSED** [review]

**Review by olgahryniuk — DISMISSED** [review]

**Review by drospa — APPROVED** [review]
Looks good to me. This last pagraph will be changed after the public testnet release.
Cardano-docs is for documentation, not for explaining timelines or release dates

**Review by olgahryniuk — APPROVED** [review]

**Pull request merged**